connexta/admin-console-beta
# DDF-2773: Fix duplicate source names being allowed

- Cannot have duplicate source names anymore
- Error will appear if attempting to create a duplicate source name
- Unit tests
- The SourceNameExists TestMethod will be called during a Persist for a specific Source Configuration Handler.

@tbatie @adimka @coyotesqrl 

Manual test:
Create a source.
Attempt to create another source with same name. Observe error message.
Test this for CSW, WFS, and OpenSearch.
